chaos_kitten/brain/orchestrator.py

```
@@ -63,88 +64,158 @@ def plan_attacks(state: AgentState) -> Dict[str, Any]:
-                endpoint.get("path"),
-            )
-            continue
+            
+        current_payloads_to_run = [{"payload": attack.get("payload"), "is_adaptive": False}]
+        
+        # Helper to run a payload and analyze it
+        async def run_single_payload(payload_val, is_adaptive=False):
+            try:
+                result = await executor.execute_attack(
+                    method=endpoint.get("method", "GET"),
+                    path=endpoint_path,
```

Ruff (0.15.1)

[warning] Function definition does not bind loop variable `endpoint_path`

(B023)